projects/components/src/table/table.component.ts

```diff
@@ -603,6 +616,10 @@ export class TableComponent
         }
       : undefined;
   }
+
+  private isColumnSortable(columnConfig: TableColumnConfig): boolean {
+    return columnConfig.sortable === undefined || columnConfig.sortable;
```

Comment:
shouldn't this be `value !== undefined && sortable` ? also, is `sortable` a valid property? shouldn't it be `sort` ?

Reply:
Actually, I wanted it to have sort by default, even if someone does not add sortable property in the column config